### ISSUE TITLE
Example does not demonstrate improved performance

### ISSUE THREAD

**Issue by Permafacture** [oswaldoludwig/Eigenvalue-Decay-Regularizer-for-Keras#3]
I am unable to use the example code to demonstrate this regularization method improving performance of the MINST model, specifically reducing over-fitting.  Since the original code did not compare the model trained with and without the regularization, I had to modify the example (Attached but with txt extension because github doesn't like .py extension).  I also swapped the training and testing set in order to exacerbate overfitting and found the regularized model performed 5% worse than the unregularized model.

Is this because 1) the regularization is not correctly implemented, 2) the model is not built or trained in a way that takes advantage of the regularization or 3) the regularization theory is not correct?

[test.txt](https://github.com/oswaldoludwig/Eigenvalue-Decay-Regularizer-for-Keras/files/3952429/test.txt)
